src/Components/Aspire.Azure.AI.OpenAI/ConfigurationSchema.json

```diff
@@ -31,14 +31,14 @@
                     "ClientOptions": {
                       "type": "object",
                       "properties": {
-                        "ApplicationId": {
```

Comment:
We need to update the readmes:

https://github.com/dotnet/aspire/blob/d0d3ce55c07baed98891a6e95e294ae6348c347e/src/Components/Aspire.OpenAI/README.md?plain=1#L75